go-controller/pkg/ovn/gateway_init.go

```diff
@@ -431,8 +433,14 @@ func addDistributedGWPort() error {
 		return nil
 	}
 
-	datapath, stderr, err := util.RunOVNSbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "datapath",
-		"external_ids:name="+ovnClusterRouter)
+	// Wait a bit for northd to create the cluster router's datapath in southbound
+	var datapath string
+	err = wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
```

Comment:
do we need to do it every 500ms?
seems to fast for an exec command and since is the setup does not seem critical to loose at least 1 sec on that

Reply:
Done